databases/week1/homework.sql

```diff
@@ -0,0 +1,37 @@
+-- 1. Find out how many tasks are in the task table
+--  Answer: 35
+SELECT COUNT(*) AS tasks_amount FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+--  Answer: 8
+SELECT COUNT(*) AS tasks_with_valid_due_amount FROM task WHERE due_date IS NULL;
+
+-- 3. Find all the tasks that are marked as done
+SELECT task.title AS done_tasks FROM task WHERE status_id = (SELECT id FROM status WHERE name='done'); -- 12 tasks
+
```

Comment:
I like your solution with the inner query. However, in this case a Join would be enough.